package.json

```diff
@@ -49,14 +49,14 @@
         "esbuild": "0.16.10",
         "eslint": "^8.14.0",
         "typescript": "^4.6.4",
-        "vite": "^5.0.0",
+        "vite": "^6.0.0",
         "vitest": "^0.34.4"
     },
     "peerDependencies": {
-        "vite": "^5.0.0"
```

Comment:
Bumped this straight to v6.

The Vite plugin is extremely stable and is feature complete.

I don't think we need to support both versions. You can stay on the current version until you are ready to update to Vite 6.

This will help keep our maintenance cost to a minimum.

Reply:
Wouldn’t that make it to version 2?

Reply:
Not according to semver.

See: https://semver.org/#what-should-i-do-if-i-update-my-own-dependencies-without-changing-the-public-api

You will need to upgrade to Vite 6 in your own project.

Reply:
Since the `laravel-vite-plugin` is not a standalone and require `vite` to work, I’d consider the drop of Vite 5 to be a breaking change.

It becomes quickly complicated for users if they have to wonder if any minor/patch update will stop to work with their current setup. I would prefer a breaking, it sends a clearer signal.

What do you think?

Reply:
just ran npm update (after v1.1.0 was released) and got this...
```
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: undefined@undefined
npm error Found: vite@5.4.11
npm error node_modules/vite
npm error   dev vite@"^5.4.10" from the root project
npm error
npm error Could not resolve dependency:
npm error peer vite@"^6.0.0" from laravel-vite-plugin@1.1.0
npm error node_modules/laravel-vite-plugin
npm error   dev laravel-vite-plugin@"^1.0.5" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
```

Reply:
I also get the same message